src/progly/query.cc

```diff
@@ -390,7 +390,7 @@ void worker()
                 int decode_runquery_cls = 0;
                 assert(decode_runquery_cls);
             }
-            nrows_processed += nrows_server_processed;
```

Comment:
I'm not sure here but as mentioned I think this should be restored to the previous accounting or in some way such that a query that matches say 3/20 rows, should report 3/3 with --use-cls flag and 3/20 without the flag.  Since the cls case should have already filtered out the rows, only the 3 matching rows are actually returned (denominator).  Without cls all 20 rows are returned from server.  The only exception is SELECT*, which does not do any accounting due to fastpath execution and hence the server does not even look at the data/metadata to determine how many rows are present, it just blindly returns the data.   Thanks.

Reply:
Done